src/components/App/App.js

```diff
@@ -30,16 +38,14 @@ class App extends Component {
         <nav>
           <h1>Rancid Tomatillos</h1>
         </nav>
-        {!this.state.movies.length && <h3>Loading...</h3>}
+        {this.state.error && <h3>{this.state.error}</h3>}
+        {!this.state.movies.length && !this.state.error && <h3>Loading...</h3>}
         {
-          this.state.clickedMovie.id ?
-          <MovieDetails key={this.state.clickedMovie.id} movieInfo={this.state.clickedMovie} updateIsClicked={this.updateIsClicked}/> :
+          this.state.clickedMovie.id && !this.state.error ?
+          <MovieDetails key={this.state.clickedMovie.id} movieInfo={this.state.clickedMovie} resetClickedMovie={this.resetClickedMovie}/> :
           <MoviesContainer 
           movies={this.state.movies} 
-          selectMovie={this.selectMovie}
-          // isClicked={this.state.isClicked} 
-          // clickedMovie={this.state.clickedMovie}
-          // updateIsClicked={this.updateIsClicked}
+          updateClickedMovie={this.updateClickedMovie}
```

Comment:
Lots of `this.state` in here - you could refactor with some destructuring, but I think you should stick with whichever you two find most comfortable.